loaders/spanner.go

```diff
@@ -324,7 +328,7 @@ func SpanTableIndexes(client *spanner.Client, table string) ([]*models.Index, er
 
 	// sql query
 	const sqlstr = `SELECT ` +
-		`INDEX_NAME, IS_UNIQUE ` +
+		`INDEX_NAME, IS_UNIQUE, INDEX_TYPE = "SEARCH" AS IS_SEARCH ` +
```

Comment:
Isn't this [document](https://cloud.google.com/spanner/docs/information-schema#indexes) applied this `"SEARCH"` INDEX_TYPE?